core/translate/plan.rs

```diff
@@ -114,6 +114,7 @@ pub enum Operator {
         table_reference: BTreeTableReference,
         predicates: Option<Vec<ast::Expr>>,
         step: usize,
+        reverse: Option<bool>,
```

Comment:
`Option<bool>` is a bit of a weird datatype for this. EIther it is reverse or it isn't :)

Maybe just an enum Direction with Forwards / Backwards?